molo/core/content_import/validation.py

```diff
@@ -13,10 +14,13 @@ def __init__(self, ws):
 
     def is_validate_for(self, locales):
         self.errors = []
+        main_language = None
         for l in locales:
             if l.get('is_main'):
                 main_language = l.get('locale')
                 break
+        if not main_language:
+            raise ImportError("No main language found")
```

Comment:
I think we still need to handle the case where multiple languages have `is_main == True` here (and test that case).